common/junit-platform-native/gradle/native-image-testing.gradle

```diff
@@ -90,6 +91,17 @@ abstract class NativeTestArgumentProvider implements CommandLineArgumentProvider
     @Input
     abstract Property<Boolean> getDiscovery()
 
+    @InputDirectory
+    abstract DirectoryProperty getExcludeConfigDir()
+
+    List<String> addExcludeConfig() {
+        String[] jarPatternPair = getExcludeConfigDir().get().file("exclude-config").getAsFile().readLines().get(0).split(",")
```

Comment:
Why is this loaded from an external file rather than being hard-coded? AFAICS this script is only applied once.

Reply:
So there is a case where users may need to add `exclude-config` manually (if they are using JUnit `5.12` and they only have dependency to `junit-platform-native` and not to `Gradle` or `Maven`). 
To help them add this exclude-config manually, I stored all necessary information in the external file, which they can just parse and use

Reply:
I see. Thanks for the explanation!